be/src/vec/olap/olap_data_convertor.cpp

```diff
@@ -222,6 +221,22 @@ Status OlapBlockDataConvertor::OlapColumnDataConvertorBitMap::convert_to_olap()
             ++bitmap_value_cur;
         }
         assert(nullmap_cur == _nullmap + _row_pos + _num_rows && slice == _slice.get_end_ptr());
+
+        slice = _slice.data();
+        raw_data = _raw_data.data();
```

Comment:
The previous code will resize the podarray when insert a bitmap everytime, it will call resize too many times, the performance is very bad.
Maybe we could compute the podarray size at the beginning by tranverse all bitmap value. And call resize only one time.
Then all the slice->data do not need re calculate any more. The code will be more simple.

Reply:
fixed